scheduleapp/templates/scheduleapp/thursday.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'homepageapp/base.html' %}
+
+{% block content %}
+
+    <h3>Thursday</h3>
```

Comment:
Please remove duplication. Those templates here should be done in generic way so you have one template only send context to it to display day, for example. There should be only one template "day.html" instead of template for every day of the week.